.github/actions/prepare-frontend/action.yml

```diff
@@ -1,10 +1,14 @@
 name: Prepare front-end environment
+inputs:
+  node-version:
+    required: false
```

Comment:
we need this to be able to pull in the node version from the build requirements. very likely the .nvmrc in the release branch will be correct, but we want a single source of truth for build requirements